frontend/shared/src/components/table/Table.tsx

```diff
@@ -198,8 +198,7 @@ TableProps<D>): React.ReactElement => {
     const headers = [
       ...(canSelectRows ? [selectorCol] : []),
       ...(canSelectOneRow ? [radioSelectorCol] : []),
-      // fix to a hard ts problem: https://github.com/TanStack/react-table/discussions/2664
-      ...columns.map((col) => ({ ...col, accessor: 'id' as keyof D })),
+      ...columns,
```

Comment:
This was a fix to this build problem: https://github.com/City-of-Helsinki/yjdh/runs/5620854899?check_suite_focus=true
Not sure how to fix the typesceipt error otherwise. Any ideas?
https://github.com/TanStack/react-table/discussions/2664

Reply:
I will try to resolve this and get back to you